webextension/background.js

```diff
@@ -1,3 +1,63 @@
+browser.contextMenus.create({
+  id: "open-in-this-container",
+  title: "Always Open in This Container",
+  contexts: ["all"],
+});
+
+function getUserContextIdFromCookieStore(cookieStore) {
+  const container = cookieStore.replace("firefox-container-", "");
+  if (container !== cookieStore) {
+    return container;
+  }
+  return false;
+}
+
+const storagePrefix = "originMap@@_";
+const storageArea = browser.storage.local;
+
+browser.contextMenus.onClicked.addListener((info, tab) => {
+  const pageUrl = new window.URL(info.pageUrl);
+  const userContextId = getUserContextIdFromCookieStore(tab.cookieStoreId);
```

Comment:
Yeah. I introduced cookieStoreId, but it's not in FF51.... we can still retrieve the userContextId from the tab as we do in index.js.